aws/data.yaml

```diff
@@ -35,13 +37,65 @@ spec:
               memory: "1Gi"
               cpu: "500m"
 ---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: data-prod
+  labels:
+    environment: prod
+    app: data
+spec:
+  selector:
+    matchLabels:
+      environment: prod
+      app: data
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        environment: prod
+        app: data
+    spec:
+      containers:
+        - name: data
+          image: docker.io/healthmapidha/dataservice:0.0.3
+          ports:
+            - containerPort: 3000
+          env:
+            - name: DB_CONNECTION_STRING
+              valueFrom:
+                secretKeyRef:
+                  name: data-prod-84t6gt4hhc
+                  key: db_connection_string
+          resources:
+            requests:
+              memory: "256Mi"
+              cpu: "100m"
+            limits:
+              memory: "1Gi"
+              cpu: "500m"
+---
```

Comment:
Any way to share common config between dev/prod? Or is it more trouble than it's worth?

Reply:
Hmm I think one can do inheritance in yaml I think but honestly I don't think we want to go this way unless we feel we get huge benefits from it...